tests/ExamplePluginWithLib/Downstream/README.md

```diff
@@ -0,0 +1,18 @@
+# Plugin sharing library - Downstream
+This example illustrates how to use a library shipped from another plugin (should be installed first).
+
+## Configure
+`cmake -DRadium_DIR=/pathToInstalledRadium/lib/cmake/Radium/  -DQt5_DIR=/pathToInstalledQt5/lib/cmake/Qt5
```

Comment:
Do Qt5 installation handled specially for this plugin ?
I don't think we need to specify that here.
```suggestion
`cmake -DRadium_DIR=/pathToInstalledRadium/lib/cmake/Radium/
```

Reply:
see my previous comment. But this will be explained differently.